packages/components/psammead-leading-story-promo/README.md

```diff
@@ -0,0 +1,61 @@
+# ⛔️ This is an alpha component ⛔️
+
+This component is currently tagged as alpha and is not suitable for production use. Following the passing of an accessibility review this component will be marked as ready for production and the alpha tag removed.
+
+# psammead-leading-story-promo - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-leading-story-promo%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-leading-story-promo%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-leading-story-promo)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-leading-story-promo) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-leading-story-promo)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-leading-story-promo&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/leading-story-promo--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-leading-story-promo.svg)](https://www.npmjs.com/package/@bbc/psammead-leading-story-promo) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `LeadingStoryPromo` component is designed to be used on 'index' pages, which are pages that link to other articles/stories. It supports having info on the left with an image on the right. This can be a collection of any nodes.
```

Comment:
I mean this is technically correct for most services, except for RTL

Reply:
Added a sentence on about LTR and RTL services in the usage section